bloom/bloom_test.go

```diff
@@ -0,0 +1,37 @@
+package bloom
+
+import (
+	"math/rand"
+	"testing"
+)
+
+// Test file for internal functions of the bloom package.
+var global4x64 [4]uint64
+
+func TestFasthash(t *testing.T) {
```

Comment:
I guess another test would be that a lot of different numbers don't all hash to the same value.

Reply:
I think it would make sense if we were "inventing" the algorithm, but in this case it's copied from the parquet specs https://github.com/apache/parquet-format/blob/master/BloomFilter.md#technical-approach, it's an internal API, and just a one-liner, so probably a bit overkill to invest more time in testing it.